src/data/README.md

```diff
@@ -16,16 +108,12 @@ TODO - describe the properties used
 
 ## `<product slug>-install.json`
 
-This file is only used in DevDot. It contains the data and content needed for each product's downloads page (`/boundary/downloads`, `/consul/downloads`, etc.).
+This file is only used in DevDot. It contains the data and content needed for each product's downloads page (`/boundary/downloads`, `/consul/downloads`, etc.). There are four possible data properties that can be provided at the highest level: `doesNotHavePackageManagers`, `featuredTutorials`, `packageManagerOverrides`, and `sidebarMarketingCard`.
```

Comment:
Added some `<details>/<summary>` elements below to make this file a little more manageable and also added Markdown comments before each to make the raw Markdown a little more manageable too.